backend/apps/pr_review_bot/tasks.py

```
@@ -19,92 +19,88 @@ def review_pr(repo: str, pr_number: int):
+
         for issue in file_issues:
-            issues.append({
-                **issue,
-                'file_path': filename
-            })
-    
+            issues.append({**issue, "file_path": filename})
+
     # LLM analysis (if enabled)
-    if getattr(settings, 'ENABLE_LLM_REVIEW', False):
+    if getattr(settings, "ENABLE_LLM_REVIEW", False):
```

Ruff (0.15.21)

[error] Undefined name `settings`

(F821)